packages/react-dom/src/events/plugins/__tests__/ChangeEventPlugin-test.js

```diff
@@ -326,32 +354,41 @@ describe('ChangeEventPlugin', () => {
     expect(called).toBe(1);
   });
 
-  it('should unmount', () => {
-    const input = ReactDOM.render(<input />, container);
+  it('should unmount', async () => {
```

Comment:
Looks like a redundant test at a glance.

Reply:
Seems like it could be cleaned up. Unless there is some special unmounting for inputs?

Reply:
Maybe. It's a pretty cheap test and doesn't really hurt us having it around just in case.